### PR TITLE
MVP-2578: "Initial fetchData" + "render" using frontendClient object (notifi-react-card)

### PR DESCRIPTION
**Pull request by eric-notifi**
1. Cosolidate ConnectedWallet in core using graphql

2. Implment copyAuthorization method in NotifiFrontendClient

3. Implement the following 3 methods  in NotifiSubscriptionContext
- `Initial` effect/ 
- `render` method/ 
- `copyAuths` method

**NOTE**
The `render` and "inital fetch" logic of frontendClient is consolidated in `NotifiSubscriptionContext` in order to reduce the structural complexity. 
Another option is creating one more hook.

### CODE REVIEW

File: packages/notifi-react-card/lib/components/subscription/NotifiSubscriptionCard.tsx [hunk 1]
```diff
@@ -77,10 +77,10 @@ export const NotifiSubscriptionCard: React.FC<
   const { params } = useNotifiClientContext();
 
   return (
-    <NotifiSubscriptionContextProvider {...params}>
-      <NotifiFormProvider>
+    <NotifiFormProvider>
+      <NotifiSubscriptionContextProvider {...params}>
```
Comment (eric-notifi) [inline packages/notifi-react-card/lib/components/subscription/NotifiSubscriptionCard.tsx:81]: Reason:
The `render` and "initial fetch" methods need to consume NotifiFormProvider

File: packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts [hunk 2]
```diff
@@ -16,6 +16,9 @@ import type {
   WalletBalanceEventTypeItem,
 } from '../models';
 import type { Authorization, NotifiStorage, Roles } from '../storage';
+import { createInMemoryStorageDriver } from '../storage';
```
Comment (Nathan-Notifi) [inline packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts:19]:
```suggestion
import { createInMemoryStorageDriver, createLocalForageStorageDriver, NotifiFrontendStorage } from '../storage';
```

File: packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts [hunk 3]
```diff
@@ -16,6 +16,9 @@ import type {
...
+import { createLocalForageStorageDriver } from '../storage';
```
Comment (Nathan-Notifi) [inline packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts:20]:
```suggestion
```

File: packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts [hunk 4]
```diff
@@ -16,6 +16,9 @@ import type {
...
+import { NotifiFrontendStorage } from '../storage';
```
Comment (Nathan-Notifi) [inline packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts:21]:
```suggestion
```

**Review by Nathan-Notifi — APPROVED** [review]

File: packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts (hunk 2 above)
Comment (eric-notifi) [reply packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts:19]: Ahh.. thanks for pointing this out

**Pull request merged**